src/pages/AppDetails.vue

```diff
@@ -148,15 +148,18 @@ function toggleMobileMenu() {
   showNotifications.value = false
 }
 
-async function createMainnetApp(app: AppConfig): Promise<AppResponse> {
+async function createMainnetApp(
+  app: AppConfig,
+  shouldCopyTestnetConfig: boolean
+): Promise<AppResponse> {
   try {
     const mainnetApp = (
       await createApp(
         {
           name: app.name,
           region: RegionMapping[selectedRegion.value.value],
```

Comment:
Remove region and default_chain if not required

Reply:
default_chain is required, region may be required, when I checked with Himank, he wasn't sure.